lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,214 @@
+              - Token used to authenticate with the API. Can also be specified via K8S_AUTH_API_KEY environment
+                variable.
+              type: str
+          username:
+              description:
+              - Provide a username for authenticating with the API. Can also be specified via K8S_AUTH_USERNAME
+                environment variable.
+              type: str
+          password:
+              description:
+              - Provide a password for authenticating with the API. Can also be specified via K8S_AUTH_PASSWORD
+                environment variable.
+              type: str
+          cert_file:
+              description:
+              - Path to a certificate used to authenticate with the API. Can also be specified via K8S_AUTH_CERT_FILE
+                environment variable.
+              type: str
+          key_file:
+              description:
+              - Path to a key file used to authenticate with the API. Can also be specified via K8S_AUTH_HOST
+                environment variable.
+              type: str
+          ssl_ca_cert:
+              description:
+              - Path to a CA certificate used to authenticate with the API. Can also be specified via
+                K8S_AUTH_SSL_CA_CERT environment variable.
+              type: str
+          verify_ssl:
+              description:
+              - "Whether or not to verify the API server's SSL certificates. Can also be specified via
+                K8S_AUTH_VERIFY_SSL environment variable."
+              type: bool
+          namespaces:
+              description:
+              - List of namespaces. If not specified, will fetch all virtual machines for all namespaces user is authorized
+                to access.
+              type: list
+          network_name:
+              description:
+              - In case of multiple network attached to virtual machine, define which interface should be returned as primary IP
+                address.
+              type: str
+    requirements:
+    - "openshift >= 0.6"
+    - "PyYAML >= 3.11"
+'''
+
+EXAMPLES = '''
+# File must be named kubevirt.yaml or kubevirt.yml
+
+# Authenticate with token, and return all virtual machines for all namespaces
+plugin: kubevirt
+connections:
+ - host: https://kubevirt.io
+   token: xxxxxxxxxxxxxxxx
+   ssl_verify: false
+
+# Use default config (~/.kube/config) file and active context, and return vms with interfaces
+# connected to network myovsnetwork and from namespace vms
+plugin: kubevirt
+connections:
+  - namespaces:
+      - vms
+    network_name: myovsnetwork
+'''
+
+from ansible.plugins.inventory.k8s import K8sInventoryException, InventoryModule as K8sInventoryModule, format_dynamic_api_exc
+
+try:
+    from openshift.dynamic.exceptions import DynamicApiError
+except ImportError:
+    pass
+
+
+API_VERSION = 'kubevirt.io/v1alpha3'
+
+
+class InventoryModule(K8sInventoryModule):
+    NAME = 'kubevirt'
+
+    transport = 'oc'
```

Comment:
This variable is never used in the code.

Reply:
Right, will remove.